examples/vagrant/chef-infra-server/provision.sh

```diff
@@ -0,0 +1,36 @@
+CHEF_SERVER_VERSION=$1
+OS_TYPE=$2
+OS_VERSION=$3
+
+echo "*************************************************************************"
+echo "Downloading & Installing Chef Server Version: $CHEF_SERVER_VERSION"
+echo "*************************************************************************"
+
+if [[ $OS_TYPE == *'ubuntu'* ]]
```

Comment:
@devopsdina Should we consider using the automate based installer instead of the OS specific packages?

Reply:
Hi @jmassardo my primary driver for this particular vagrantfile was to test chef server upgrades and migrations.  If the automate installer can install older version of the chef-server, I think that should be ok.  I couldn't find in the docs how to pass a chef server version though, I was looking at: https://automate.chef.io/docs/infra-server/.  Do you know, if for example, I could pass chef server version 12.17.15 or 12.3.0 with the automate installer?

Eventually, I'd like to add another vagrantfile for the automate installer in these examples too.

Reply:
@devopsdina Gotcha. That makes sense. Can you add a note to the readme with some details about the use case?